types/markdown-it/package.json

```diff
@@ -16,8 +16,8 @@
         }
     },
     "dependencies": {
-        "@types/linkify-it": "*",
-        "@types/mdurl": "*"
+        "@types/linkify-it": "^5",
+        "@types/mdurl": "^2"
```

Comment:
These look correct to me given upstream's dependencies. 
![Uploading image.png…]()